plugins/org.eclipse.mat.api/src/org/eclipse/mat/inspections/LeakHunterQuery.java

```diff
@@ -99,7 +99,7 @@
 @HelpUrl("/org.eclipse.mat.ui.help/tasks/runningleaksuspectreport.html")
 public class LeakHunterQuery implements IQuery
 {
-
+    private final static boolean HIDECL = Boolean.getBoolean("org.eclipse.mat.inspections.leakhunter.hidecl");

```

Comment:
Would it be better to set the property with `@Argument(flag="....")`. I think they can be passed from the command line arguments then?

For example, https://github.com/eclipse-mat/mat/blob/dbee70361757980ef8391f20c2bdde51935ab947/plugins/org.eclipse.mat.api/src/org/eclipse/mat/inspections/ObjectListQuery.java#L44-L45 provides a boolean parameter `inbound`

Reply:
Arguments can be used for headless MAT but is there a way to set arguments so that they're always used for GUI MAT to override default values? With just a vanilla `@Argument`, the user would have to specify it each time (and most users run leak suspects from the wizard or from the Overview pane which wouldn't offer a chance to set arguments, so they'd have to run it from the Query Browser).